src/main/resources/segmentation.yaml

```diff
@@ -0,0 +1,742 @@
+        maxScaleFactor: 1.2
+        yOffset: 0.0
+        pitchVariability: 0.1
+        density: 0.2
+      - texture:
+           diffuse: "/textures/fern2_rotated.png"
+           normal: null
+        numBoards: 4
+        xScale: 1.2
+        yScale: 1.2
+         # overall scalar for the model
+        scale: 0.7
+        minScaleFactor: 0.75
+        maxScaleFactor: 1.2
+        yOffset: 0.0
+        pitchVariability: 0.1
+        density: 0.2
+   fallenLeaves:
+      density: 1.0
+      scale: 0.7
+      minScaleFactor: 0.75
+      maxScaleFactor: 1.2
+      yOffset: 0.0
+      pitchVariability: 0.1
+
+lighting:
+   # Strength of the ambient lighting in the scene
+   ambientStrength: 0.4
+   # Power to use in the specular calcualtion
+   specularPower: 16
+   hdr:
+      enabled: true
+      # The exposure value to use in the tone mapping algorithm (larger = more light)
+      exposure: 0.6
+   gammaCorrection:
+      enabled: true
+      # The gamma value to use for gamma correction
+      gamma: 2.2
+   sun:
+      # Display a polygon representation of the sun
+      display: false
+      # Automatically determine the sun's position from the sky HDR
+      autoPosition: true
+      # Number of sides to use in the polygon
+      numSides: 10
+      # Strength (and colour) of sun to use in lighting calculations
+      strength: [ 3.8, 3.3, 3.2 ]
+      # Position of the sun polygon and for use in lighting calculations (Currently overwritten by the HDR image)
+      position: [ 50.0, 200.0, -50.0 ]
+      # Scale of the sun polygon
+      scale: 20.0
+   sky:
+      # Path to HDR file
+      hdrFile: "textures/gamrig_8k.hdr"
+      # Resolution of generated cubemap face textures
+      resolution: 2048
+   ssao:
+      enabled: true
+      # Number of samples in the kernel
+      kernelSize: 32
+      # Radius of samples
+      radius: 0.5
+      # Bias for depth test
+      bias: 0.025
+   translucency:
+      enabled: true
+      # Scalar for leaf translucency (larger = brighter impact from translucency)
+      factor: 0.1
+   shadows:
+      enabled: true
+      # Resolution of the shadow map (larger scenes will need a larger resolution to avoid pixelation artefacts/missing details at the cost of GPU memory)
+      resolution: 4096
+      # Bias to avoid shadow acne, reduce if shadows are detached from objects, increase if quantization errors (stripes) appear
+      bias: 0.005
+   volumetricScattering:
+      enabled: true
+      # Number of samples along each light ray (larger = higher resolution)
+      numSamples: 100
+      # Density of samples along each light ray (higher = samples taken from a smaller area = shorter, brighter rays)
+      sampleDensity: 1.3
+      # Impact decay of each sample (lower = greater fall-off)
+      decay: 0.99 # range [0.0, 1.0)
+      # Scalar for brightness of each ray (higher = brighter)
+      exposure: 0.0015
+      # Brightness value to clamp samples at
+      maxBrightness: 100.0
+
```

Comment:
Any parameters with default values don't need to be specified in this file and should be removed for clarity. Similarly with segmentationRandom.yaml. Also `src/main/resources/default.yaml` needs updated to match the README (include new segmentation parameters)

Reply:
Default values removed from `segmentation.yaml` and `segmentationRandom.yaml`. I've also now updated `default.yaml` to match the README.